hip/static/js/index.js

```diff
@@ -25,5 +25,20 @@ import RightScrollNav from "../../../apps/hip/static/js/rightScrollNav";
 document.addEventListener("DOMContentLoaded", function() {
   Header();
   SideBar();
-  RightScrollNav();
+  lazyLoad();
+
+  function lazyLoad() {
```

Comment:
Do we need this function? Can't we just do `lazyLoadRightScrollNav()` on line 28?

Reply:
I don't think, I'm happy to remove it. I just did this in case there are other components that need to be lazy loaded in the future. I rather that they all in there on "lazy load" function than just in lazyLoad code block.

Reply:
Hmm, even better would be for the lazy loading to happen within `RightScrollNav()` itself, rather than having `lazyload` in the name anywhere. The component is the best place to determine whether it should load since it has the knowledge of knowing what it needs to be able to function properly.

Reply:
done